x-pack/plugins/observability_solution/observability_ai_assistant/server/routes/chat/route.ts

```diff
@@ -96,6 +96,12 @@ const chatCompleteRoute = createObservabilityAIAssistantServerRoute({
         conversationId: t.string,
         title: t.string,
         responseLanguage: t.string,
+        instructions: t.array(
```

Comment:
Can this not be an array of strings?

Reply:
It could but then it won't be able to override instructions set in the knowledge base, options:

1. Read instructions from Knowledge base **only** if no instructions were passed via the API
2. Allow duplicate / conflicting instructions (by just merging two simple string arrays)
3. Keep the ID to merge the two sets, giving API params priority (current)

Reply:
Right, I remember now. Let's do 4, it will keep the API simple for basic use cases, WDYT?

Reply:
Fixed in https://github.com/elastic/kibana/pull/180263/commits/b70178f5d0245729d15317cfc4fedadc95ba1926